package/kernel/linux/modules/leds.mk

```
@@ -16,6 +16,27 @@ define KernelPackage/leds-gpio
   AUTOLOAD:=$(call AutoLoad,60,leds-gpio,1)
 endef
 
+
+define KernelPackage/led-group-multicolor
+	 SUBMENU:=$(LEDS_MENU)
+	 TITLE:=LEDs group multi-color support
+	 KCONFIG:= \
+	     CONFIG_LEDS_CLASS_MULTICOLOR=y \
+	     CONFIG_LEDS_GROUP_MULTICOLOR
+	 DEPENDS:=@LINUX_6_6
+	 FILES:=$(LED_TRIGGER_DIR)/rgb/leds-group-multicolor.ko
```

I do not think this is working.
I think you need this:
```suggestion
	 FILES:=$(LINUX_DIR)/drivers/leds/rgb/leds-group-multicolor.ko
```

---

I have done the changes but I will have to test build when I get home